foreman/foreman.spec

```diff
@@ -199,6 +199,8 @@ BuildRequires: %{?scl_prefix}rubygem(autoprefixer-rails) >= 5.2
 BuildRequires: %{?scl_prefix}rubygem(autoprefixer-rails) < 6.0
 BuildRequires: %{?scl_prefix}rubygem(patternfly-sass) >= 2.7
 BuildRequires: %{?scl_prefix}rubygem(patternfly-sass) < 3.0
+BuildRequires: %{?scl_prefix}rubygem(sprockets) >= 3
+BuildRequires: %{?scl_prefix}rubygem(sprockets) < 4
```

Comment:
`~> 3` is equivalent to `>= 3` really.  Is that a mistake in Foreman's Gemfile?  If not, this line should be removed.